src/components/charts/ChartGroupDetails.tsx

```diff
@@ -131,7 +131,6 @@ export default function ChartGroupDetails() {
                 component={DeleteComponentsName.ChartGroup}
                 redirectTo={true}
                 url={`${URLS.CHARTS}/discover`}
-                reload={false}
```

Comment:
is it intentional?

Reply:
Yes vivek, reload is a function, but here false is passed, have tested after removing it also working fine